src/components/layout/header/header.tsx

```diff
@@ -1,8 +1,19 @@
+import { ROUTE_PATH } from '@/constant/routes';
+import useLogoutMutate from '@/hooks/useLogoutMutate';
+import { removeLocalStorage } from '@/utils/storage';
 import { Box, Button, Toolbar, Typography } from '@mui/material';
 import AppBar from '@mui/material/AppBar';
+import { useNavigate } from 'react-router-dom';
 
 const Header = () => {
-  const handleLogout = () => {};
+  const navigate = useNavigate();
+  const logoutMutation = useLogoutMutate();
```

Comment:
요렇게 리액트 쿼리의 훅 이름을 그대로 따라주는게 좋지 않을까요?
```suggestion
  const logoutMutation = useLogoutMutation();
```